src/pages/assets.tsx

```diff
@@ -275,6 +282,24 @@ const AssetsPage = ({ data }: PageProps<Queries.AssetsPageQuery, Context>) => {
           />
         </Row>
 
+        <Row>
+          <AssetDownload
+            title={translateMessageId("page-assets-leslie-the-rhino", intl)}
+            alt={translateMessageId("page-assets-leslie-the-rhino", intl)}
+            artistName="Tomo Saito"
+            artistUrl="https://tomosaito.com/"
+            image={data.leslieTheRhino}
+          />
+          <AssetDownload
+            title={translateMessageId("page-assets-leslie-the-rhino", intl)}
+            alt={translateMessageId("page-assets-leslie-the-rhino", intl)}
+            artistName="Tomo Saito"
+            artistUrl="https://tomosaito.com/"
+            image={data.leslieTheRhino}
+            shouldHide={true}
```

Comment:
Is this just for left aligning the image container? Seems weird but looks like we already implemented it...

Reply:
Without it the image is full width and huge. I just used another example of this from where the wallet robot is added.